.github/ISSUE_TEMPLATE/kubernetes-onboarding-template.md

```diff
@@ -54,27 +55,20 @@ _*The responsible parties are listed below each item in the checklist_
 _Infrastructure Team_
-- [ ] **AWS Elastic Container Registry (ECR) repository for the app container:** _provide name of ECR repo here_  
-_Infrastructure Team_ | [PRs welcome](https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/global/ecr.tf)  
-- [ ] **Automation to release and tag the app's GitHub repo with a semantic version number** | [Example](https://github.com/department-of-veterans-affairs/vsp-infra-calico/blob/main/.releaserc) | [Docs](https://semantic-release.gitbook.io/semantic-release/)    
-_Requesting team; Infrastructure Team can assist_  
+- [ ] **AWS Elastic Container Registry (ECR) repository for the app container:** Submit PR to add your application name to the "all-repos" section of [this page]((https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/global/ecr.tf))  
+_Requesting team_  
+- [ ] **Add semantic release configuration (a `.releaserc` file) to your repo** | [Example](https://github.com/department-of-veterans-affairs/vsp-infra-calico/blob/main/.releaserc) | [Docs](https://semantic-release.gitbook.io/semantic-release/)    
+_Requesting team_  
 - [ ] **Automation to push the app's container to ECR with a semantic version number** | [Example](https://github.com/department-of-veterans-affairs/vsp-infra-calico/blob/main/.github/workflows/mirror-images.yaml)    
-Note: Don't use default "latest" tag. The release system uses modified container image tags to synchronize automation. 
-_Requesting team; Infrastructure Team can assist_  
-- [ ] **Kubernetes manifest using Helm charts** in `vsp-infra-application-manifests` | [Example](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/vsp-amt/grafana) | [Argo CD Docs](https://argo-cd.readthedocs.io/en/stable/)  
-Note: Manifests _can_ be specified in several formats, however, we are standardizing around [Helm charts](https://helm.sh/docs/topics/charts/).  
-_Requesting team; Infrastructure Team can assist_  
-- [ ] **Kubernetes detect and update application** in `vsp-infra-applications-manifests` | [Environments](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/vsp-operations/argocd-apps)  
-_Requesting team; Infrastructure Team can assist_  
-- [ ] **Automation to update the Kubernetes manifest** when a new version of the app's container is pushed to ECR | [Example](https://github.com/department-of-veterans-affairs/vsp-infra-calico/blob/main/.github/workflows/update-manifests.yaml)  
-_Requesting team; Infrastructure Team can assist_
+Note: We recommend using the `sha`. Don't use default "latest" tag. The release system uses modified container image tags to synchronize automation. 
```

Comment:
This is where I mentioned the git sha vs semantic release thing. Would really need to check with Kyle on which one we want to enforce or if we want to allow a choice or if it matters at all... I am not sure the answer because we have working examples/patterns of both right now... can still approve and we can make a note to revisit.

Reply:
I will get @ph-One's feedback on this and update next week if necessary.

Reply:
Allow both -- a `sha` or semver. Leave it up to the application owner, but we absolutely need to guide them away from tag reuse, such as `latest`.

Reply:
https://media.giphy.com/media/3o85xF9vVInNI6KAlG/giphy.gif

Reply:
https://media.giphy.com/media/3o85xF9vVInNI6KAlG/giphy.gif

Reply:
![Uploading giphy.gif…]()